lib/devtools/config.rb

```diff
@@ -150,7 +150,7 @@ class Mutant < self
     # Devtools configuration
     class Devtools < self
       FILE = 'devtools.yml'.freeze
-      DEFAULT_UNIT_TEST_TIMEOUT = 0.1  # 100ms
+      DEFAULT_UNIT_TEST_TIMEOUT = 0.1 # 100ms
```

Comment:
@dkubb I think our convention is contradicted by this rubocop related change. Can you confirm, if so we should disable this cop. //cc @backus